src/main/webapp/app/entities/organization/organization.service.ts

```diff
@@ -37,6 +40,14 @@ export class OrganizationService {
     }
 
     findCurrent(): Observable<EntityResponseType> {
-        return this.http.get<IOrganization>(`${this.resourceUrl}/current`, { observe: 'response' });
+        const result = this.http.get<IOrganization>(`${this.resourceUrl}/current`, { observe: 'response' });
+        result.subscribe(res => {
+            this.storage.store(USER_MODE_STORAGE_KEY, res.body.userMode);
```

Comment:
The value for existing and not newly created orgs is written as `null` to local storage.
```suggestion
            this.storage.store(USER_MODE_STORAGE_KEY, res.body.userMode || UserMode.TEAM);
```